environments/production/terraform.yml

```diff
@@ -162,7 +162,7 @@ servers:
       volume_size: 2800
     group: "elasticsearch"
     os: jammy
-    count: 2
+    count: 1
```

Comment:
We should remove one more machine from `es_a3{i}-production`. We want to ensure that both azs have the same number of machines.